### ISSUE TITLE
[macOS] clicking to correct is difficult to do

### ISSUE THREAD

**Issue by campionfellin** [campionfellin/CLOxIssueTracker#4]
The clickboxes do not seem to register all the time

**Comment by rsquizz** [comment]
Note: this got better after I upgraded to Big Sur but (I think) before I downloaded the newest release of CLOx.

**Comment by campionfellin** [comment]
Yeah, I haven't changed anything in CLOx related to this recently... glad it worked better on an OS upgrade

**Comment by campionfellin** [comment]
I'm thinking about getting rid of this, especially with larger transcripts it can cause some memory issues. What do you think? / in the future maybe bringing it back as a separate tab like "upload transcript, edit, redownload"

**Comment by rsquizz** [comment]
I get that we don't have a huge user base, but I am a little hesitant about taking an existing feature out. Would it make sense if we limit the length of the audio file that this feature works for? It could either be a hard restriction based on file size or transcript length or a soft restriction in the form of a warning that files over a certain size may cause problems.

I say all of this without knowing what the code looks like, so if it's going to be a pain from a development point of view, then let's take it out for now.

**Comment by campionfellin** [comment]
I think there's a couple improvements - I can first (a) refactor this section and see if it improves. Right now, each word is its own element, and so then it's rendering hundreds / thousands of new elements and (b) if that still doesn't help then yeah I'll just put a limit on it
